### ISSUE TITLE
tests hangs on TestBasicUsage_N10N sometimes

### ISSUE THREAD

**Issue by host6** [voedger/voedger#359]
- [x] #470

https://github.com/voedger/voedger/actions/runs/5454579310/jobs/9924890449?pr=357
https://github.com/voedger/voedger/actions/runs/5499064472/jobs/10020994906
hangs rarely:
```golang
func TestXxx(t *testing.T) {
	Test_Race_n10n_perSubject(t)
	Test_Race_n10nCHS(t)
	Test_Race_n10nCHSU(t)
	Test_Race_SUsignUpIn(t)
	TestBasicUsage_N10N(t)
}

func TestBasicUsage_N10N(t *testing.T) {
	require := require.New(t)
	vit := it.NewVIT(t, &it.SharedConfig_Simple)

	ws := vit.WS(istructs.AppQName_test1_app1, "test_ws")

	n10nChan := vit.SubscribeForN10n(ws, it.QNameTestView)

	// call test update to the view
	body := fmt.Sprintf(`
 		{
 			"App": "%s",
 			"Projection": "my.View",
 			"WS": %d
 		}`, istructs.AppQName_test1_app1.String(), ws.WSID)
	vit.Post("n10n/update/13", body)

	offset := <-n10nChan
	log.Println(offset)

	// the cannel will be closed automatically on vit.TearDown()
	vit.TearDown()
	_, ok := <-n10nChan
	require.False(ok)
}
```
```[tasklist]
### Tasks
- [ ] https://github.com/voedger/voedger/issues/518
- [ ] https://github.com/voedger/voedger/issues/528
```

**Comment by host6** [comment]
Probably fixed by #470 and https://github.com/untillpro/airs-router2/commit/e43fd07266c35b24a2d2341117f0e507c6754e55
Will watch over.

**Comment by host6** [comment]
still occurs: https://github.com/voedger/voedger/actions/runs/5668722145/job/15359998071?pr=508
